corgi/web/templates/base_manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+    "creationInfo": {
+        "created": "{% now 'Y-m-d' %}T{% now 'H:i:00' %}Z",
+        "creators": [
+            "Organization: Red Hat Product Security (secalert@redhat.com)"
+        ],
+        "licenseListVersion": "3.8"
+    },
+    "dataLicense": "CC-BY-4.0",
+    "documentDescribes": [
+        "SPDXRef-{{obj.uuid}}"
+    ],
+    "documentNamespace": "https://access.redhat.com/security/data/manifest/spdx/{{obj.name}}-{{obj.uuid}}",
+    "name": "{{obj.name}}",
+    {% block packages %}{% endblock %}
```

Comment:
Minor nit: I see we have e.g. the relationships key below in this document, and then the included template only iterates over the relationships. But we have the packages key in the included template - can we move it up to this template instead?

Reply:
I don't think we can move the packages blocks up to the base_manifest. Both product_manifest.json and component_manifest.json have different packages blocks. The product_manifest one relates each component to it's subcomponents whereas the component one only has a single layer of nesting, so only subcomponents in the one component we are describing.

Reply:
Sorry, I should have added an example. I'm looking at the logic for the packages block in this template:
```
{% block packages %}{% endblock %}    
```

Which is different from the relationships block:
```
    "relationships": [
       {% block relationships %}{% endblock %}
    ],
```

Let's add the key (not the values in the array) to this template so they're both consistent:
```
    "packages": [
       {% block packages %}{% endblock %}
    ],
```

Reply:
Ok, I'll add the packages array declaration to the base template and include the packages block inside it.